### ISSUE TITLE
[False positive] Speed

### ISSUE THREAD

**Issue by Janmm14** [konsolas/AAC-Issues#1]
Speed sometimes gives false positives with default sprinting speed of .35
Vanilla clients are being flagged when moving sometimes up to .42 difference when the expected diff was .35.

A video will follow.

**Comment by CeIebrimbor** [comment]
Logs too

**Comment by Janmm14** [comment]
No item speed, just a local 1.8 test server using 1.9.10-b6 default config with verbose.

**Comment by Janmm14** [comment]
I do have this block setup of a slightly raising terrain. (each ```_``` is one block)
```
___
______
_________
_____________
________________
```
Sprintjumping downwards causes `moved too quickly midair` violations with ```f:sprintinghvelxxxx phasexx``` and a block diff of up to ~0.36 while ~0.337 was expected. (other following violations do have lower differences like ~0.35 instead of ~0.334, ~0.324 instead of ~0.321, ~0.328 instead of ~0.326)

Also occasionally glide is being detected, saying `didn't fall correctly` when strafing edges of blocks while jumping downwards.

Video will follow some time, just have to upload it.

Violations also occurr sprintjumping upwards with `f:grounded sprinting`
Moved ~0.401-0.406 instead of ~0.350 or a movement of ~0.379-0.392 instead of ~0.350
Evaluating minimums and maximums of a short testing session (15 mins)

Sprintjumping around a bit I found out earlier that its possible to trigger up tp .42 difference when the allowed was .35 or lower.
